lib/ProductOpener/Food.pm

```diff
@@ -1229,49 +1327,34 @@ sub compute_nutrition_score ($product_ref) {
 		]
 	);
 
-	$product_ref->{misc_tags} = ["en:nutriscore-not-computed"];
+	return;
+}
 
-	my $prepared = '';
+=head2 is_nutriscore_applicable_to_the_product_categories($product_ref)
+
+Check that the product has a category, that we know if it is a beverage or not,
+and that it is not in a category for which the Nutri-Score should not be computed
+(e.g. food for babies)
```

Comment:
Baby food: It would be nice to send/use the exemption debug message for knowledge panels